mllib/src/test/scala/org/apache/spark/ml/clustering/GaussianMixtureSuite.scala

```diff
@@ -267,6 +268,18 @@ class GaussianMixtureSuite extends MLTest with DefaultReadWriteTest {
     assert(trueLikelihood ~== floatLikelihood absTol 1e-6)
   }
 
+  test("GMM support instance weighting") {
```

Comment:
The final model is highly sensitive to the initlization, which is computed in `initRandom` and highly related to the data distribution & partition.